src/Store/SyntaxHighlightingStoreConnector.re

```diff
@@ -17,11 +17,10 @@ module NativeSyntaxHighlights = Oni_Syntax.NativeSyntaxHighlights;
 module Protocol = Oni_Syntax.Protocol;
 
 // TODO:
-// - Move effects to Service_Terminal
 // - Move updater to Feature_Terminal
 // - Change subscription granularity to per-buffer -
 // - this could help remove several effects!
-let start = (languageInfo: Ext.LanguageInfo.t) => {
+let start = (cli: Core.Cli.t, languageInfo: Ext.LanguageInfo.t) => {
```

Comment:
This seems too coarse-grained. Does the syntax highlighting service really need to know about the CLI interface?

Reply:
Switched to just use an `enabled` flag